package io.fiap.erp.exception;

public class UsuarioJaExistenteException extends RuntimeException {

    public UsuarioJaExistenteException(){}

    public UsuarioJaExistenteException(String message){}

    public UsuarioJaExistenteException(String message, Throwable cause){}

    public UsuarioJaExistenteException(Throwable cause){}

}
